### DISCUSSION TITLE
Helf for the Turbo Transfer

### DISCUSSION THREAD

**Discussion by TheKimac84** [gopro/OpenGoPro#170] [Q&A] [answered]
Hello, I wanted to have some information about the "Turbo Transfer" feature of the GoPro open API.
When I run the command "/gopro/media/turbo_transfer?p=1" the camera marks me on its screen "transfer in progress" but I have no way of knowing where on my computer the files are going. Also, is it possible to choose where we want the transfer to go (choose as a destination)?

**Comment by KonradIT** [comment, marked as answer]
Turbo transfer doesn't send any file anywhere, just tweaks the camera HTTP server to serve files in larger buffers. You need to query the files you want to download manually.
